content/redirect/cli-command-requires-channel.adoc

```diff
@@ -0,0 +1,6 @@
+---
+layout: redirect
+redirect_url: https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+CLI
```

Comment:
The specific documentation for this does not currently exist on the wiki. I still think it's important to create a specific stub redirect now to allow us to add that documentation in the future.

That said, with minor changes (just mention that some commands/command invocations cannot work in SSH CLI), https://jenkins.io/doc/book/managing/cli/#using-the-cli-client could be it.